### PR TITLE
Mongo atlas

### PR DESCRIPTION
**Pull request by blarghmatey**
Adding deployment of MongoDB Atlas clusters for use with Open edX installations

### CODE REVIEW

File: src/ol_infrastructure/infrastructure/mongodb_atlas/__main__.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+#############
+# VARIABLES #
+#############
+environment_name = f"{stack_info.env_prefix}-{stack_info.env_suffix}"
+target_vpc = network_stack.require_output(env_config.require("target_vpc"))
+business_unit = env_config.get("business_unit") or "operations"
+aws_config = AWSBase(tags={"OU": business_unit, "Environment": environment_name})
+max_disk_size = atlas_config.get("disk_autoscale_max_gb")
+max_instance_type = atlas_config.get("cluster_autoscale_max_size")
+min_instance_type = atlas_config.get("cluster_autoscale_min_size")
```
Comment (shaidar) [inline src/ol_infrastructure/infrastructure/mongodb_atlas/__main__.py:33]: Don't see where this is defined

Comment (blarghmatey) [reply src/ol_infrastructure/infrastructure/mongodb_atlas/__main__.py:33]: It's not defined for the stacks that I've set up, just exposing it as an option in the code.

File: src/ol_infrastructure/infrastructure/mongodb_atlas/__main__.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            description="Access to Mongodb cluster from Vault",
+        ),
+    ],
+    egress=default_egress_args,
+)
+
+atlas_aws_network_peer = atlas.NetworkPeering(
+    f"mongo-atlas-network-peering-{environment_name}",
+    accepter_region_name=target_vpc["region"],
+    container_id=atlas_cluster.container_id,
```
Comment (shaidar) [inline src/ol_infrastructure/infrastructure/mongodb_atlas/__main__.py:91]: Is this right? The docs looks like it's one word

**Comment by shaidar** [comment]
👍

**Pull request merged**